README.md

```diff
@@ -1,8 +1,8 @@
 <h1 align="center">
-        <a href="https://www.lightdash.com">⚡️ Lightdash </a>
+        <a target="_blank" href="https://www.lightdash.com"><img align="center" style="max-width:300px;" src="/static/screenshots/lightdash_logo_banner.png"> </a>
```

Comment:
@TuringLovesDeathMetal We can either put in a light mode logo like below: 

![Github Inverted Logo (1)](https://user-images.githubusercontent.com/14341285/142236083-40aca6f1-a410-47fa-accc-285d152114e5.png)

or stick to one with a coloured background that works on both: 
![OpenGraph Image (3)](https://user-images.githubusercontent.com/14341285/142236274-04b0200d-3cb0-46b9-819e-a14df1364060.png)